tests/ballerina-unit-test/src/test/resources/test-src/balo/test_balo/types/finite_type_test.bal

```diff
@@ -88,11 +88,15 @@ function finiteAssignmentRefValueTypeCaseTwo() returns foo:POrInt {
 }
 
 function testFiniteTypeWithMatch() returns foo:PreparedResult {
-    match foo() {
-        foo:PreparedResult x => return x;
-        () => return "qqq";
-        error => return "qqq";
+    var x = foo();
+    if (x is foo:PreparedResult) {
+        return x;
+    } else if (x is ()) {
```

Comment:
can every check from this point be in a single `else` block?

Reply:
yes  +1